postgres/postgres.go

```diff
@@ -129,7 +130,7 @@ func chkInvalidIdentifier(identifer ...string) bool {
 // WhereByRequest create interface for queries + where
 func WhereByRequest(r *http.Request, initialPlaceholderID int) (whereSyntax string, values []interface{}, err error) {
 	whereKey := []string{}
-	whereValues := []string{}
+	whereValues := []interface{}{}
```

Comment:
if you use FormatArray instead pq.Array this variable don't need to be []interface{} can be []string